elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -176,111 +225,127 @@
+        border-bottom: 1px solid #{pfe-color(surface--border--lightest)}; // use a var here
     }
 }
 
 @mixin pfe-trigger-expanded {
     position: relative;
     display: block;
+    border-bottom: 0px;
 }
 
 @mixin pfe-trigger-color-light {
-    --pfe-expanded--main:         transparent;
-    --pfe-expanded--aux:          #{pfe-color(surface--lightest--text)};
-    --pfe-expanded--focus:        #{pfe-color(surface--lightest--link--focus)};
+    --pfe-accordion--main:         transparent;
+    --pfe-accordion--aux:          #{pfe-color(surface--lightest--text)};
+    --pfe-accordion--focus:        #{pfe-color(surface--lightest--link--focus)};
+    --pfe-accordion--border-top-bottom: #{pfe-color(surface--border--lightest)} solid 1px;
 }
 
 @mixin pfe-trigger-color-light-expanded {
-    --pfe-expanded--main:         #{pfe-color(surface--lightest)};
-    --pfe-expanded--aux:          #{pfe-color(surface--lightest--text)};
-    --pfe-expanded--focus:        #{pfe-color(surface--lightest--link--focus)};
-    --pfe-expanded--border-left:  #{pfe-color(ui-link)};
+    --pfe-accordion--main:              #{pfe-color(surface--lightest)};
+    --pfe-accordion--aux:               #{pfe-color(surface--lightest--text)};
+    --pfe-accordion--focus:             #{pfe-color(surface--lightest--link--focus)};
+    --pfe-accordion--border-left:  solid 3px #{pfe-color(surface--lightest--link)};
+    --pfe-accordion--border-right: solid 1px #{pfe-color(surface--border--lightest)};
+}
+@mixin pfe-trigger-color-lightest {
+  --pfe-accordion--main:         #{pfe-color(surface--lightest)};
+  --pfe-accordion--aux:          #{pfe-color(surface--lightest--text)};
+  --pfe-accordion--focus:        #{pfe-color(surface--lightest--link--focus)};
+  --pfe-accordion--border-top-bottom: #{pfe-color(surface--border--lightest)} solid 1px;
 }
 
+//// DARK
+
 @mixin pfe-trigger-color-dark {
-    --pfe-expanded--main:         transparent;
-    --pfe-expanded--aux:          #{pfe-color(surface--darkest--text)};
-    --pfe-expanded--focus:        #{pfe-color(surface--darker--link--focus)};
+    --pfe-accordion--main:         transparent;
+    --pfe-accordion--aux:          #{pfe-color(surface--darkest--text)};
+    --pfe-accordion--focus:        #{pfe-color(surface--darker--link--focus)};
 }
 
 @mixin pfe-trigger-color-dark-expanded {
-    --pfe-expanded--main:         #{pfe-color(surface--darker)};
-    --pfe-expanded--aux:          #{pfe-color(surface--darkest--text)};
-    --pfe-expanded--focus:        #{pfe-color(surface--darker--link--focus)};
-    --pfe-expanded--border-left:  #{pfe-color(surface--lightest)};
+    --pfe-accordion--main:         #{pfe-color(surface--darkest)};
+    --pfe-accordion--aux:          #{pfe-color(surface--darkest--text)};
+    --pfe-accordion--focus:        #{pfe-color(surface--darkest--link--focus)};
+    --pfe-accordion--border-left:  solid 3px #{pfe-color(surface--darkest--link)};
```

Comment:
@starryeyez024 One small change: right now the header border is blue on a dark background, it should be white.